ste/xfer-anyToRemote.go

```diff
@@ -259,9 +264,15 @@ func scheduleSendChunks(jptm IJobPartTransferMgr, srcPath string, srcFile common
 				if chunkReader != nil {
 					_ = chunkReader.Close()
 				}
+
+				// Copy the error so that we can avoid the prefetch err being overwritten under us.
```

Comment:
Doesn't the code above stop it from ever being changed?  In which case, why introduce this func here, to protect us from changes in something that will no longer change?

And if we did have to copy the error, why not do this (simpler than making separate func)

copyOfError := prefetchErr
cf = createSendToRemoteChunkFunc(jptm, id, func() { jptm.FailActiveSend("chunk data read", copyOfError ) })

Reply:
It _shouldn't_ be overwritten. I'll remove this.